test/unit/region.spec.js

```diff
@@ -122,18 +122,17 @@ describe('region', function() {
         onSwapOut: function() {}
       });
 
-      this.MyView = Backbone.View.extend({
+      this.MyView = Marionette.View.extend({
```

Comment:
ah yeah it is a bit confusing in the pits of test land, but this change has a few side effects as to coverage of non Mn Views in regions :fire: